ssv/spectest/tests/runner/pre_consensus_justifications/prev_valid_pre_consensus.go

```diff
@@ -0,0 +1,120 @@
+package pre_consensus_justifications
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/ssv/spectest/tests"
+	"github.com/bloxapp/ssv-spec/types"
+	"github.com/bloxapp/ssv-spec/types/testingutils"
+)
+
+// PreviousValidPreConsensus tests pre-consensus justification after receiving a valid quorum of pre-consensus
+func PreviousValidPreConsensus() tests.SpecTest {
```

Comment:
The name implies the quorum of preconsensus messages are for a prvious slot?
But I guess I got the implication wrong because I don't see it in the code

Reply:
changed the name to be more precise